index.html

```diff
@@ -36,7 +37,129 @@ <h1>I’m Alexandra.<br>Glad to see you!</h1>
                             <li><a href="#"><img src="img/github.jpg" alt="github icon"></a></li>
                             <li><a href="#"><img src="img/angellist.jpg" alt="angellist icon"></a></li>
                         </ul>
-        </section>  
+        </section>
+    </section>  
+            <!-- the first card............... -->
+            <section id="work-section">
+                <div class="cards">
+                    <img src="img2/Snapshoot Portfolio.png" alt="portfolio img">
+                    <div class="align-text">
+                    <h2 class="h2-size">Tonic</h2>
+                    <ul class="first-list">
+                        <li class="list1">CANOPY</li>
+                        <li class="list2">Back End Dev</li>
+                        <li class="list2">2015</li>
+                    </ul>
+                    <P class="work-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+                    <ul class="blue-list">
+                        <li>HTML</li>
+                        <li>Css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button class="work-btn btn">see project</button>
+                    </div>
+                </div>
```

Comment:
Well done on the cards so far 💯 but your card is lacking some details to match the Figma design, the border, for example, and also the background of the tech stack. Take a look at the differences 👇🏽 

## Your design

![Screenshot 2023-04-30 at 00 19 21](https://user-images.githubusercontent.com/45739963/235335335-727ff3ac-5547-48c5-adad-71f869a0a0e5.png)

## Figma

![Screenshot 2023-04-30 at 00 20 03](https://user-images.githubusercontent.com/45739963/235335354-727a7a94-69cb-4d65-a3de-d6d4e615cf1d.png)